bluesky/utils.py

```diff
@@ -1080,3 +1086,52 @@ def __call__(self, status_objs_or_none):
 def _L2norm(x, y):
     "works on (3, 5) and ((0, 3), (4, 0))"
     return np.sqrt(np.sum((np.asarray(x) - np.asarray(y))**2))
+
+
+def merge_cycler(cyc):
```

Comment:
Suggested docstring:

```
Specify movements of sets of interdependent axes atomically.

Inspect the keys of ``cyc`` (which are Devices) to indentify those which are
interdependent. Merge those speciously independent entries into one entry.
```

and then show a simple example with two interdependent keys and one independent key.